packages/keyring-eth-ledger-bridge/src/ledger-keyring.test.ts

```diff
@@ -1024,6 +1069,55 @@ describe('LedgerKeyring', function () {
         );
       });
 
+      it('resolves properly when message domain salt is an ArrayBuffer', async function () {
```

Comment:
Since the initial bug was to also cover `salt` as a `string` (if I understood correctly), shouldn't we have a test for this as well?